BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Help/DigitalInvoiceHelpViewController.swift

```diff
@@ -59,7 +59,7 @@ final class DigitalInvoiceHelpViewController: UIViewController {
         scrollView.addSubview(contentView)
         contentView.addSubview(stackView)
 
-        let backButtonTitle = NSLocalizedStringPreferredFormat("ginibank.digitalinvoice.help.backToInvoice",
+        let backButtonTitle = NSLocalizedStringPreferredGiniBankFormat("ginibank.digitalinvoice.help.backToInvoice",
                                                                comment: "Digital Invoice")
```

Comment:
[Suggestion] same suggestion: a indentation would be nice here, to align `comment` parameter in the right place